tests/rules/no-invalid-named-grid-areas.test.js

```diff
@@ -0,0 +1,288 @@
+/**
+ * @fileoverview Tests for no-invalid-named-grid-areas rule.
+ * @author xbinaryx
+ */
+
+//------------------------------------------------------------------------------
+// Imports
+//------------------------------------------------------------------------------
+
+import rule from "../../src/rules/no-invalid-named-grid-areas.js";
+import css from "../../src/index.js";
+import { RuleTester } from "eslint";
+import dedent from "dedent";
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+	plugins: {
+		css,
+	},
+	language: "css/css",
+});
+
+ruleTester.run("no-invalid-named-grid-areas", rule, {
+	valid: [
+		".grid { grid-template-areas: 1fr / auto 1fr auto; }",
+		'.grid { grid-template-areas: "a a a" "b b b"; }',
+		'.grid { grid-template-areas: " a a a " "b b b"; }',
+		'.grid { GRID-TEMPLATE-AREAS: "a a a" "b b b"; }',
+		dedent`
+        .grid { grid-template-areas: "head head"
+                                     "nav  main"
+						             "nav  foot"; 
+        }`,
+		dedent`
+        .grid { grid-template-areas: "head      head"
+                                     "nav       main"
+						             "nav       foot"; 
+        }`,
+		dedent`
+        .grid { grid-template-areas: 
+                /* "" */ "head head"
+                "nav  main" /* "a b c" */
+				"nav  foot" /* "" */;
+        }`,
+		".grid { grid-template-areas: none; }",
+		".grid { grid-template-areas:   none; }",
+		".grid { grid-template-areas: NONE; }",
+		".grid { grid-template-areas: /*comment*/ none /* comment */; }",
+		".grid { grid-template-areas: /*comment*/ NONE /* comment */; }",
+		'.grid { grid-template-areas: /* comment "" " */ none; }',
+		dedent`
+        .grid {
+            grid-template-areas: /* "comment" */ none /* "comment " */;
+        }`,
+		dedent`
+        .grid {
+			grid-template:
+				"a a a" 40px
+				"b c c" 40px
+				"b c c" 40px / 1fr 1fr 1fr;
+		}`,
+	],
+	invalid: [
+		{
+			code: 'a { grid-template-areas: "a a a" "b b"; }',
+			errors: [
+				{
+					messageId: "unevenGridArea",
+					line: 1,
+					column: 34,
+					endLine: 1,
+					endColumn: 39,
+				},
+			],
+		},
+		{
+			code: 'a { grid-template-areas: "a a a" "b b a"; }',
+			errors: [
+				{
+					messageId: "nonRectangularGridArea",
+					data: { name: "a" },
+					line: 1,
+					column: 34,
+					endLine: 1,
+					endColumn: 41,
+				},
+			],
+		},
+		{
+			code: 'a { grid-template-areas: "a a" "a ."; }',
+			errors: [
+				{
+					messageId: "nonRectangularGridArea",
+					data: { name: "a" },
+					line: 1,
+					column: 32,
+					endLine: 1,
+					endColumn: 37,
+				},
+			],
+		},
```

Comment:
Can we format it like https://github.com/eslint/css/pull/169#discussion_r2160075776 to clearly see which row is actually being reported

```css
grid-template-areas: 
     "a a"
      "a .";
```

It should report the 2nd row.